akka-stream-testkit/build.sbt

```diff
@@ -1,8 +1,6 @@
 import akka._
-import com.typesafe.tools.mima.plugin.MimaKeys
 
 AkkaBuild.defaultSettings
 Formatting.formatSettings
 OSGi.streamTestkit
 Dependencies.streamTestkit
```

Comment:
But wasn't that for the 2.3 - 2.4 compatiblity? Within 2.4.x we don't want to detect accidental change in testkit.